Source/Project64/Multilanguage.h

```diff
@@ -88,12 +88,10 @@ enum LanguageStringID{
 	MENU_LANGUAGE		=	175,
 
 	MENU_HELP			=	180,
-		MENU_USER_MAN	=	181,
-		MENU_GAME_FAQ	=	182,
-		MENU_ABOUT_INI	=	183,
-		MENU_ABOUT_PJ64	=	184,
-		MENU_FORUM   	=	185,
-		MENU_HOMEPAGE	=	186,
+		MENU_ABOUT_INI	=	181,

```

Comment:
it is fine, but you did not have to change the values for: MENU_ABOUT_INI, MENU_FORUM, MENU_ABOUT_PJ64, MENU_HOMEPAGE.

I assume you went through every language file to make sure you have renumbered the strings